Cargo.toml

```diff
@@ -29,13 +29,15 @@ keywords = ["gamedev", "cgmath", "collision"]
 [lib]
 name = "collision"
 
-[features]
-eders = ["serde", "serde_derive", "cgmath/serde"]
-
 [dependencies]
 num = "0.1"
 rand = "0.3"
 approx = "0.1"
-cgmath = "0.15"
+serde_derive = "1.0"
 serde = { version = "1.0", optional = true }
-serde_derive = { version = "1.0", optional = true }
\ No newline at end of file
+
+[target.'cfg(features = "serde")'.dependencies]
+cgmath = { version = "0.15", features = ["serde"] }
```

Comment:
What is wrong with specifying it in feature as `cgmath/serde`?

Reply:
Can't be done, feature declaration is implicit.

Reply:
See this gitter conversation: https://gitter.im/slide-rs/specs?at=59cd1719bac826f0540d947f